### PR TITLE
Scroll token claim flow

### PR DESCRIPTION
**Pull request by mcgingras**
tbd wip
opening early for visibility

### CODE REVIEW

**Comment by jefag** [comment]
I didn't see this PR until after I did #459.

Instead of reconciling, I think it makes sense to get the prisma working in isolation, merge to main, and then you remove that from your PR.

**Comment by jefag** [comment]
...reason being is we want the scroll schema into the repo, before the token claim flow is merged.

File: package-lock.json
Comment (stepandel) [inline package-lock.json:1]: pls don't commit `package-lock.json` ty!

File: package.json [hunk 1]
```diff
@@ -57,8 +59,11 @@
     "class-variance-authority": "^0.6.1",
     "clsx": "^1.2.1",
     "connectkit": "^1.5.3",
+    "cookies-next": "^4.2.1",
     "csstype": "^3.0.10",
+    "csv-writer": "^1.6.0",
```
Comment (mcgingras) [inline package.json:64]: can remove some of these when we merge, these are just for the script, which Jeff is going to rewrite

**Pull request closed without merging**